staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -103,6 +103,7 @@ type KubeSchedulerConfiguration struct {
 
 	// PluginConfig is an optional set of custom plugin arguments for each plugin.
 	// Omitting config args for a plugin is equivalent to using the default config for that plugin.
+	// +listType=atomic
```

Comment:
Does it make any difference? in terms of codegen or some other aspect

Reply:
I'm not so sure, but it seems to be a new API rule